cmd/nri-kubernetes/main.go

```diff
@@ -214,17 +213,21 @@ func createIntegrationWithHTTPSink(httpServerPort string) (*integration.Integrat
 	return integration.New("com.newrelic.kubernetes", "test-ksm", integration.Writer(h))
 }
 
-func getK8sConfig(tryLocalKubeConfig bool) (*rest.Config, error) {
-	c, err := rest.InClusterConfig()
+func getK8sConfig(tryLocalKubeConfig bool, c config.Mock) (*rest.Config, error) {
+	inclusterConfig, err := rest.InClusterConfig()
 	if err == nil || !tryLocalKubeConfig {
-		return c, nil
+		return inclusterConfig, nil
 	}
```

Comment:
I do not get the use case we are covering here: Is there a reason for not trying local kubeconfig in all cases?

Reply:
mmm no real reason, it was a piece of code migrated from the old config